tests/functional/test_query.py

```diff
@@ -29,9 +29,9 @@ def test_basic_query(chain, eth_tester_provider):
         "num_transactions",
         "number",
         "parent_hash",
-        "size",
```

Comment:
removed because potentially this is an expensive lookup and based on the comment, basic queries we want to avoid expensive lookups (transactions missing, etc.)